integreat_cms/static/src/js/pois/opening-hours/component/opening-hours-display.tsx

```diff
@@ -25,8 +26,8 @@ const OpeningHoursDisplay = ({ openingHours, setSelectedDays, translations, days
                     key={translations.weekdays[day]}
                     title={translations.editWeekdayLabel}
                     className="flex flex-wrap justify-between gap-2 hover:bg-gray-50 p-4 border-b cursor-pointer"
```

Comment:
```suggestion
                    className={cn("flex flex-wrap justify-between gap-2 p-4 border-b", {
                        "hover:bg-gray-50 cursor-pointer": canChangeEvent,
                    })}
```
Hmm, I think the hover effect and the pointer cursor somehow arise the impression that there is something to be clicked, so I'd remove the classes without the edit permissions.
This requires the import `import cn from "classnames";`.